src/services/redisSessionStorage.ts

```diff
@@ -593,30 +662,53 @@ export default class RedisSessionStorage extends RedisStorageUtils
           resolve(this.arrayStringReply(err, response))
         );
       });
+      // Deserialize all available user payloads and skip invalid one
+      const errorOrDeserializedUsers = errorOrUserWalletTokens.map(_ =>
```

Comment:
why it's called 'errorOrUserWalletTokens' if it's just the whole session payload ?
(shouldn't this be named 'errorOrSerializedUser' ?

Reply:
fixed in c4e4093